docs/docusaurus/docs/snippets/e2e_pandas_s3/03-create-expectations-programatically.py

```diff
@@ -0,0 +1,30 @@
+import great_expectations.expectations as gxe
+from great_expectations import get_context
+
+# TODO: will become from great_expectations import get_context, ExpectationSuite
```

Comment:
Should be fixed as a followup to https://greatexpectations.atlassian.net/browse/V1-67